test/views/RegistrationSentViewSpec.scala

```diff
@@ -25,7 +25,7 @@ class RegistrationSentViewSpec extends ViewBehaviours with ImplicitDateFormatter
 
   private val messageKeyPrefix = "registrationSent"
   private val section: Option[String] = Some(messages("declaration.section"))
-
+  private val firstLinkContent: String = "javascript:window.print()"
```

Comment:
could you please remove this reference its not used inactula method pageWithPrintOrDownloadLink - url

Reply:
hi, @abhijeetardale, do you want it removed, or add an assertion that checks the value?

Reply:
Assertion added.